brili.ts

```diff
@@ -754,18 +754,18 @@ function evalInstr(instr: bril.Instruction, state: State): Action {
   }
 
   case "char2int": {
-    let code = getChar(instr, state.env, 0).charCodeAt(0);
-    let val = BigInt.asIntN(64, BigInt(code));
+    let code = getChar(instr, state.env, 0).codePointAt(0);
+    let val = BigInt.asIntN(64, BigInt(code as number));
     state.env.set(instr.dest, val);
     return NEXT;
   }
 
   case "int2char": {
-    let i = getInt(instr, state.env, 0)
-    if(i > 65535 || i < 0) {
+    let i = getInt(instr, state.env, 0);
+    if(i > 1114111 || i < 0) {
       throw error(`value ${i} cannot be converted to char`);
-    }
-    let val = String.fromCharCode(Number(i));
+    } 
+    let val = String.fromCodePoint(Number(i));
```

Comment:
I maintained the range because `fromCodePoint` throws if the value is not a valid code point